src/Ultz.DF2/CommandSender.cs

```diff
@@ -21,9 +21,9 @@ public void SendEnd()
                 return;
             }
 
-#if DEBUG
```

Comment:
CoreSendEvent and CoreReceiveEvent are only used by the test suite, and should be excluded from literally any other scenario because it adds nothing and the performance is abysmal for large commands; hence the preprocessor directives.

Reply:
I used `[Conditional]` on them so they'll be removed at callsite when debug is off